editor/plugins/org.fusesource.ide.wsdl2rest.ui/pom.xml

```diff
@@ -26,9 +26,9 @@
 		<version.commons.validator>1.6</version.commons.validator>
 		<!-- TODO: Adjust this wsdl2rest version to the final Fuse productized 
 			version when it is available -->
-		<version.wsdl2rest>0.8.0.fuse-720034</version.wsdl2rest>
-		<version.commons.collections>3.2.1</version.commons.collections>
-		<version.commons.lang>2.4</version.commons.lang>
+		<version.wsdl2rest>0.8.0.fuse-730094-redhat-00001</version.wsdl2rest>
+		<version.commons.collections>3.2.2</version.commons.collections>
+		<version.commons.lang>3.8.1</version.commons.lang>
```

Comment:
Why 3.8.1? It seems that wsdl2rest is using version 3.5:

![image](https://user-images.githubusercontent.com/1105127/57068279-8b260480-6cd1-11e9-8393-e47f1c102f56.png)